src/Helper/WebformHelper.php

```diff
@@ -13,24 +15,11 @@ final class WebformHelper {
   /**
    * Get template context.
    */
-  public function getTemplateContext(WebformSubmissionInterface $webformSubmission, CprServiceResult $cprServiceResult, array $configuration = []) {
-
-    $elements = [];
-    $blacklistedElements = $configuration['blacklist_elements_for_template'] ?? [];
-    $submissionData = $webformSubmission->getData();
+  public function getTemplateContext(WebformSubmissionInterface $webformSubmission, CprServiceResult $cprServiceResult, EntityTypeManagerInterface $entity_type_manager, Renderer $renderer, array $configuration = []) {
```

Comment:
Rather than passing an `EntityTypeManagerInterface` and a `Renderer` to this function, I think we should just add it to the `WebformHelper` constructor.